src/ansys/mapdl/core/_commands/post26/set_up.py

```diff
@@ -0,0 +1,1668 @@
+        information, see `Postprocessing Nodal-Averaged Results
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_elem/Hlp_E_CH2_2.html#>`_
+
+        **Coordinate systems:** Generally, element nodal quantities stored by :ref:`ansol` are obtained in
+        the solution coordinate system ( :ref:`rsys`, SOLU) and then averaged. There are some exceptions as
+        listed below. :ref:`ansol` does not transform results from :ref:`rsys`,SOLU (or from the coordinate
+        systems described for the exceptions below) to other coordinate systems. Verify that all elements
+        attached to the subject node have the same coordinate system before using :ref:`ansol`.
+
+        * Layered element results are in the layer coordinate system ( :ref:`rsys`,LSYS). You can further
+          specify the element nodal results, for some elements, with the :ref:`shell`, :ref:`layerp26`, and
+          :ref:`force` commands.
+
+        * When :ref:`ansol` is used to store nodal-averaged result data (based on the ``DataType`` setting),
+          the global Cartesian coordinate system ( :ref:`rsys`,0) is used.
+
+        **Shell elements:** The default shell element coordinate system is based on node ordering. For shell
+        elements the
+        adjacent elements could have a different :ref:`rsys`,SOLU, making the resultant averaged data
+        inconsistent. A message to this effect is issued when :ref:`ansol` is used in models containing
+        shell elements. Ensure that consistent coordinate systems are active for all associated elements
+        used by the :ref:`ansol` command.
+
+        **Derived quantities:** Some of the result items supported by :ref:`ansol` ( :ref:`ANSOL_tab_1`) are
+        derived from the component quantities. Issue :ref:`avprin` to specify the principal and vector sum
+        quantity averaging methods.
+
+        **Default:** If ``Mat``, ``Real``, and ``Ename`` are not specified, all elements attached to the
+        node are considered. When a material ID, real constant ID, or element-type discontinuity is detected
+        at a node, a message is issued. For example, in a FSI analysis, a ``FLUID30``element at the
+        structure interface would be considered; however, because it contains no SX result, it is not used
+        during :ref:`store` operations.
+
+        .. _ANSOL_tab_1:
+
+        ANSOL - General Result Item and Component Labels
+        ************************************************
+
+        .. flat-table:: General Item and Component Labels :ref:`ansol`, ``NVAR,NODE,Item,Comp,Name,Mat,Real,Ename,DataType``
+           :header-rows: 1
+
+           * - Item
+             - Comp
+             - Description
+           * - :rspan:`3` S
+             - X, Y, Z, XY, YZ, XZ
+             - Component stress. This item stores `nodal-averaged results<https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_elem/Hlp_E_CH2_2.html#nodeavepost>`_ ifthey are available on the results file.
```

Comment:
Missing space

```suggestion
             - Component stress. This item stores `nodal-averaged results <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_elem/Hlp_E_CH2_2.html#nodeavepost>`_ ifthey are available on the results file.
```

Reply:
Fixed in https://github.com/ansys/pyconverter-xml2py/pull/374/commits/ba14c6e591ee3105313988d403f809484d812579